src/index.js

```diff
@@ -164,12 +164,22 @@ window.TurbolinksAnimate = window.TurbolinksAnimate || new function() {
         return;
       }
 
-      newElement.style[cssPropertyToCamelCase(property)] = getComputedStyle(element).getPropertyValue(property);
+      let properties = property.split(',');
+      for (var i = 0; i < properties.length; i++) {
```

Comment:
We might want to consider using `forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) with an arrow function here. It looks a bit cleaner to me.

```suggestion
      properties.forEach(() => {
```

Reply:
Much cleaner!